lib/devise/passwordless/mailer.rb

```diff
@@ -6,6 +6,7 @@ class Mailer < Devise::Mailer
     def magic_link(record, token, remember_me, opts = {})
       @token = token
       @remember_me = remember_me
+      @opts = opts
```

Comment:
Can you help me understand the need for this? `opts` is currently used as a passthrough to Devise for things like setting email headers. I take it you are sending a magic link and need to render some data in the view template that you can't get through the user/resource model? In that case, if it were easier to attach extra metadata to the `@token` itself would that solve your use case?